azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/AbstractAzureMojo.java

```diff
@@ -441,7 +445,7 @@ public String getAuthMethod() {
     public void execute() throws MojoExecutionException {
         try {
             // init proxy manager
-            ProxyUtils.initProxy(Optional.ofNullable(this.session).map(s -> s.getRequest()).orElse(null));
+            ProxyUtils.initProxy(Optional.ofNullable(this.session).map(MavenSession::getRequest).orElse(null));
```

Comment:
do we have to use ProxyUtils? why not config proxy in `Azure.az().config()`?

Reply:
will fix in future PR